integration-tests/docker/test_env/test_env_builder.go

```diff
@@ -308,6 +310,43 @@ func (b *CLTestEnvBuilder) Build() (*CLClusterTestEnv, error) {
 					b.te.LogStream.SaveLogLocationInTestSummary()
 				}
 				b.l.Info().Msg("Finished shutting down LogStream")
+
+				if b.t.Failed() {
```

Comment:
@Tofel should we always dump or only if some flag in config is set?

Reply:
I didn't want to add more flags, so I always dump